helm/single-content/values.yaml

```
@@ -125,6 +126,13 @@ ingress:
 
   # -- Ingress controller path for index connections.
   indexPath: "/index/"
+  # -- Whether to expose the index connection through ingress
+  exposeIndexPort: true
+
+ # -- Ingress controller path for service connections.
+  servicePath: "/content/service/"
+  # -- Whether to expose the service connection through ingress
+  exposeServicePort: true
```

Don't think we need the expose parameters - just make the schema accept empty values and test accordingly before creating paths in ingress
```suggestion
 # -- Ingress controller path for service connections. Empty string to disable.
```